frame/contracts/src/wasm/mod.rs

```diff
@@ -149,66 +125,82 @@ impl ExportedFunction {
 	}
 }
 
-impl<T: Config> PrefabWasmModule<T> {
-	/// Create the module by checking and instrumenting `original_code`.
+/// Costs for operations that are related to code handling.
+#[cfg_attr(test, derive(Debug, PartialEq, Eq))]
+#[derive(Clone, Copy)]
+enum CodeToken {
+	/// Weight for loading a contract per byte.
+	Load(u32),
+}
```

Comment:
what's the point of adding an enum with 1 variant here?

Reply:
yeah, once we removed the `Reinstrument` variant, this can really become a newtype struct